README.md

```diff
@@ -88,14 +88,16 @@ Example inspec.yml:
 name: my-profile
 title: My own AWS profile
 version: 0.1.0
-inspec_version: '>= 2.2.7'
+inspec_version: '>= 4.6.9'
 depends:
   - name: inspec-aws
-    url: https://github.com/inspec/inspec-aws/archive/master.tar.gz
+    url: https://github.com/inspec/inspec-aws/archive/x.y.z
```

Comment:
I think the naming of this is still being discussed (and will be pinned down in a future PR) but it will certainly end in `.tar.gz`

Reply:
I changed the ending just in case.